### ISSUE TITLE
What this line will do? I can see that you are declaring css rules for 2 class selector and 1 html element. Is these rules are applicable for all three?

### ISSUE THREAD

**Issue by sajid73** [Kowshik-18/Leader_Board_CSS-3#3]
https://github.com/Kowshik-18/Leader_Board_CSS-3/blob/1ec5fa48242c2ec6703fc3ef18a1f7358ca0e05e/styles/style.css#L118

**Comment by Kowshik-18** [comment]
No, This rule applies only  **img (tag)** which is internal of  this two-class **course  & course-banner** 

These will work in this sequence:
 **1. class course >> 2. class course-banner  >> 3. img (tag)**

**Comment by sajid73** [comment]
Then why not write only 
.course-banner img{

???

**Comment by Kowshik-18** [comment]
> Then why not write only
> .course-banner img{
> 
> ???


You can also use this and meaning is the same but

**I use this to simply understand my css-code that which tag/class is inner in which tag/class**

**Comment by sajid73** [comment]
I think it's not a good practice. Because, it may twist css rules with other elements, Which may inside of first selector.
